### DISCUSSION TITLE
unreachable code after return statement

### DISCUSSION THREAD

**Discussion by macellinoss275** [livewire/livewire#6434] [7. Bugs (archived)]
### Livewire version

3.0@beta

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Fresh install Laravel 10
2. Install Livewire Version 3.x

### Additional information

I just created a brand new Laravel project and then installed Livewire 3 after that. Then I created a simple layout and components just to test wire:navigate. And there are 2 warnings whenever I go to any page (whether it's my first time or after wire:navigate) like this:
![image](https://github.com/livewire/livewire/assets/138418079/c9eeb040-577f-4a42-a99d-995602acba61)

And if I click the warning, it shows this:
![image](https://github.com/livewire/livewire/assets/138418079/30cc289f-1b13-450a-a024-26240c2ff82d)

If this matter, here are my specs:
- PHP 8.2.9
- Node.js 18.17.1 LTS
- Composer 2.5.8
- Vite 4.0.0
- Laravel 10.10
- Livewire 3.0@beta
- Windows 11
- Visual Studio Code
- Firefox Browser latest version

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by delta-source** [comment]
Same issue for me.

Node v18.17.0
PHP 8.2.8
Laravel 10.20.0
Composer 2.5.8
Livewire 3.0

**Comment by Khal-ID** [comment]
Did anyone solve this issue? I know it's just a warning but I'm facing it right now. Would be nice to get rid of it.

PS. I'm not even using wire:navigate. I just upgraded from ^v2.5

Laravel 10.22.0
PHP 8.2.9
Livewire 3.0.2
Node 18.17.1
Composer 2.6.2

**Comment by silvershades** [comment]
Same error here

**Reply by Khal-ID** [reply]
It's just a warning, not an error.

**Comment by kresnasatya** [comment]
I'm facing this issue too although this doesn't make error in my Laravel app. Is that mean that Livewire execute two times?

**Reply by Khal-ID** [reply]
If it was running twice you should get a different warning. This is related to some code Livewire is trying to reach but can't due to a return statement. I wouldn't worry tbh.

**Comment by diegoanso** [comment]
I also got this issue, but it looks like is on purpose [https://github.com/livewire/livewire/discussions/6065](url). I'll be commenting the code after the return for now.

**Reply by kresnasatya** [reply]
I get 404 link. Could you please repair the link?

**Comment by code-reflex** [comment]
Even I am facing this issue when I am adding a livewire component into my filament PHP application.  The associated CSS is not loading  in my case and hence I started looking for errors/warnings in the console and saw these errors.  

Any guidance to resolve this issue is appreciated.
